portfolio/src/App.jsx

```diff
@@ -209,12 +197,11 @@ function App() {
             </div>
           </div>
         </section>
+
         {/* Additional skills section*/}
-        <section>
```

Comment:
Figured I would add an ID to this section so that you could include it in the header if you wanted to link to it like you're doing with the other sections